src/Insect/Parser.purs

```diff
@@ -236,6 +237,7 @@ normalUnitDict = Dictionary
   , Q.fortnight ==> ["fortnights", "fortnight"]
   , Q.month ==> ["months", "month"]
   , Q.year ==> ["years", "year"]
+  , Q.julianYear ==> ["julianYears", "julianYear"]
```

Comment:
I initially added `julianyear` and `julianyears` to this list, but the unit is "prettified" in the REPL as `julianYear`, and having both `julianyear` and `julianYear` is a very bad experience for completion.